### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by piuccio**
@janua

### CODE REVIEW

File: project/dependencies.scala [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  val awsSdk = "com.amazonaws" % "aws-java-sdk-s3" % "1.10.37"
+  val awsSdk = "com.amazonaws" % "aws-java-sdk-s3" % "1.10.47"
   val commonsIo = "org.apache.commons" % "commons-io" % "1.3.2"
-  val contentApi = "com.gu" %% "content-api-client" % "7.13"
+  val contentApi = "com.gu" %% "content-api-client" % "7.17"
   val mockito = "org.mockito" % "mockito-all" % "1.10.19" % "test"
   val playJson = "com.typesafe.play" %% "play-json" % "2.4.4"
   val scalaTest = "org.scalatest" %% "scalatest" % "2.2.5" % "test"
-  val specs2 = "org.specs2" %% "specs2" % "2.3.12" % "test"
+  val specs2 = "org.specs2" %% "specs2" % "3.7" % "test"
```
Comment (janua) [inline project/dependencies.scala]: I tried this and there seems to be no `specs2` `3.7` for scala `2.10.x`. Could you try it locally?

Comment (piuccio) [inline project/dependencies.scala]: not sure how, I have scala 2.11 locally

Comment (piuccio) [inline project/dependencies.scala]: apparently it [goes up to](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22specs2_2.10%22) `3.3`

Comment (janua) [inline project/dependencies.scala]: `++2.10.5` to set it to `2.10`.

`+test` to cross scala version test.

Comment (piuccio) [inline project/dependencies.scala]: I've reverted the change on test, it's not really important

**Comment by janua** [comment]
:+1: 
It's already merged.

**Pull request merged**